src/Tracim.jsx

```diff
@@ -52,77 +47,50 @@ export const Tracim = () => {
     return (
       <View style={styles.pageContainer}>
         <StatusBar />
-
-        {IS_SINGLE_SERVER
-          ? (
-            <SingleServerMenu
-              onPressServer={navigation.navigate}
-            />
-          ) : (
-            <MultipleServerMenu
-              serverList={serverList}
-              onPressServer={navigation.navigate}
-              onPressAdd={(server) => setServerList([...serverList, server])}
-              onPressRemove={(server) => setServerList(serverList.filter(s => s.url !== server.url))}
-            />
-          )}
-      </View>
-    )
-  }
-
-  function WebViewScreen({ route, navigation }) {
-    return (
-      <View style={styles.pageContainer}>
-        <StatusBar />
-        <SafeAreaView style={styles.pageContainer}>
-          <WebView
-            onClickGoBack={() => navigation.goBack()}
-            screenId={route.params.screenId}
-            url={route.params.url}
-          />
-        </SafeAreaView>
-        {!IS_SINGLE_SERVER && (
-          <TouchableHighlight
-            style={styles.openServerMenuButton}
-            onPress={() => navigation.openDrawer()}
-          >
-            <Icon icon={faServer} />
-          </TouchableHighlight>
-        )}
+        <MultipleServerMenu
+          serverList={serverList}
+          onPressServer={(server) => { navigation.navigate(server.name)}}
+          onPressAdd={(server) => setServerList([...serverList, server])}
+          onPressRemove={(server) => setServerList(serverList.filter(s => s.url !== server.url))}
+        />
       </View>
     )
   }
 
   return (
-    <NavigationContainer>
-      <Drawer.Navigator
-        initialRouteName={t('Home')}
-        screenOptions={{
-          drawerActiveTintColor: 'white',
-          drawerActiveBackgroundColor: COLORS.PRIMARY,
-          drawerPosition: 'right'
-        }}
-      >
-        <Drawer.Screen
-          component={HomeScreen}
-          name={t('Home')}
-          options={{
-            headerShown: false,
-            swipeEnabled: !IS_SINGLE_SERVER
-          }}
-        />
-        {serverList.map(server => <Drawer.Screen
-          component={WebViewScreen}
-          initialParams={{ url: server.url }}
-          key={`drawer_${server.name}`}
-          name={server.name}
-          options={{
-            headerShown: false,
-            swipeEnabled: !IS_SINGLE_SERVER
+    serverList.length > 0 && (
```

Comment:
Why was this condition added? If the application is in multi server mode, it can have 0 elements in the list.

Reply:
To be able to initialize`initialRouteName` to the server in single server mode. But the test is indeed missing `!IS_SINGLE_SERVER`.

Reply:
Ok, I see the problem with `serverList[0].name`, I'm gonna change the condition